schema/workflow.yaml

```diff
@@ -830,28 +837,8 @@ $defs:
     properties:
       with:
         title: WithEvent
-        type: object
+        $ref: '#/$defs/eventProperties'
```

Comment:
This one could be a `OneOf` of expression, pattern, or `eventProperties`.

Reply:
I'm not sure about that one. @cdavernas @fjtirado @matthias-pichler-warrify wdyt ?